src/audio/copier/copier.c

```diff
@@ -995,14 +966,6 @@ static int copier_comp_trigger(struct comp_dev *dev, int cmd)
 
 	for (i = 0; i < cd->endpoint_num; i++) {
 		if (dev->ipc_config.type == SOF_COMP_HOST && !cd->ipc_gtw) {
-			ret = comp_set_state(cd->endpoint[i], cmd);
-			if (ret < 0)
-				break;
-
-			if (ret == COMP_STATUS_STATE_ALREADY_SET) {
-				ret = PPL_STATUS_PATH_STOP;
-				break;
-			}
```

Comment:
why is this removed in this commit?

Reply:
because this part is for host dev, since host dev removed, there is no need to set host dev state and related check.